### ISSUE TITLE
Create filter category for "Tools providing accessibility information"

### ISSUE THREAD

**Issue by vavroom** [w3c/wai-eval-tools#51]
In the filter list, the last item is "Tools providing accessibility information". This filter does not currently have a category associated with it. It lives below the "License" category. This is somewhat confusing as some people may interpret it as being a type of license.

A category should be created for this filter. Perhaps something along the lines of "Tool accessibility".

![screen shot 2017-11-13 at 10 10 07](https://user-images.githubusercontent.com/29165258/32732686-5f6481d0-c85b-11e7-9258-02e5a9520315.png)
![screen shot 2017-11-13 at 10 10 17](https://user-images.githubusercontent.com/29165258/32732688-5f75b8ba-c85b-11e7-82a4-e463d2a14747.png)

**Comment by yatil** [comment]
Hi @vavroom, we wanted to have this featured without a category so that it is highlighted and easy to access (thus encouraging tool vendors to provide that information in the first place). Probably we need a different visual treatment.

**Comment by yatil** [comment]
(Assigned Shadi for FYI, no immediate need to comment.)
